desc/equilibrium/equilibrium.py

```diff
@@ -64,6 +64,8 @@ class Equilibrium(IOAble, Optimizable):
         total toroidal flux (in Webers) within LCFS. Default 1.0
     NFP : int (optional)
         number of field periods Default ``surface.NFP`` or 1
+    NFP_umbilic_factor : int (optional)
+        field period prefactor Default ``surface.NFP_umbilic_factor`` or 1
```

Comment:
this is not a very good description, I would say "integer dividing the field period, such that the effective field period is NFP/NFP_umbilic_factor" or something.

Also out of curiosity, would this work with NFP that is not equal to one?